DigitalOffice/Services/ProjectService/Broker/Responses/IUserExistenceResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace LT.DigitalOffice.ProjectService.Broker.Responses
+{
+    /// <summary>
+    /// Interface for a respond that shows whether the user exists or not.
+    /// </summary>
+    public interface IUserExistenceResponse
+    {
+        bool Exists { get; }
```

Comment:
Why not just use a boolean variable in IOperationResult?

Reply:
I wasn't sure if we would want to modify the response in the future

Reply:
Use IOperationResult